ci_scripts/flake8_diff.sh

```diff
@@ -1,156 +1,3 @@
 #!/bin/bash
 
-# Inspired from https://github.com/scikit-learn/scikit-learn/blob/master/build_tools/travis/flake8_diff.sh
```

Comment:
I had left the original script in-tact originally since i wasn't sure if we wanted to remove everything (e.g. shortcutting if no new commits). But I would prefer this way as well, it's much easier to parse and flake8 is not the bottleneck in our CI procedure anyway.

Reply:
The new script is even faster because it does not invoke any git commands.

Reply:
Makes sense 👍